### ISSUE TITLE
Content blocks sorting does not work

### ISSUE THREAD

**Issue by bmihelac** [django-oscar/django-oscar#2162]
oscar/apps/dashboard/promotions/views.py
Line 93

```
    def post(self, request, **kwargs):
        """
        When called with a post request, try and get 'promo[]' from
        the post data and use it to reorder the page content blocks.
        """
        data = dict(request.POST).get('promo[]')
```

It should use `promo`

I see, that in https://github.com/django-oscar/django-oscar/pull/1859/files new `jquery-sortable-min.js` is introduced, and similar change is made in:

https://github.com/django-oscar/django-oscar/pull/1859/files#diff-39607f84727e7c78aee2e95c553bad6aL203

patch coming soon

**Comment by sasha0** [comment]
Indeed. Thanks for the report!

**Comment by sasha0** [comment]
@bmihelac would you like to create a PR with your patch 8425e6a referenced above? Thanks!

**Comment by bmihelac** [comment]
@sasha0 sure, I'll make it ASAP

**Comment by bmihelac** [comment]
Added PR:
https://github.com/django-oscar/django-oscar/pull/2247

sorry but I do not know easy way to test this change.

**Comment by sasha0** [comment]
Fixed in 8425e6ad1c229659e7014a778d985ef3cacb88d7.
